packages/atomic/cypress/integration/facets/segmented-facet/segmented-facet.cypress.ts

```diff
@@ -29,22 +36,124 @@ describe('Segmented Facet Test Suites', () => {
+      FacetAssertions.assertNumberOfSelectedBoxValues(1);
+      FacetAssertions.assertNumberOfIdleBoxValues(defaultNumberOfValues - 1);
+    });
+    describe('verify analytics', () => {
+      before(setupSelectSegmentedFacet);
+      FacetAssertions.assertLogFacetSelect(field, selectionIndex);
+    });
+  });
+
+  describe('with custom #numberOfValues', () => {
+    const customNumValues = 20;
+    function setupCustomSegmentedFacet() {
+      new TestFixture()
+        .with(
+          addSegmentedFacet({
+            field: 'language',
+            label: 'Language',
+            'number-of-values': customNumValues,
+          })
+        )
+        .init();
+    }
+
+    describe('verify rendering', () => {
+      before(setupCustomSegmentedFacet);
+
+      CommonAssertions.assertAccessibility(segmentedFacetComponent);
+      CommonFacetAssertions.assertLabelContains(
+        SegmentedFacetSelectors,
+        'Language'
+      );
+      FacetAssertions.assertNumberOfIdleBoxValues(customNumValues);
+    });
   });
 
-  describe.skip('with custom #numberOfValues', () => {});
+  describe.skip('when no search has yet been executed', () => {
+    before(() => {
+      new TestFixture()
+        .with(addSegmentedFacet({field, label}))
+        .withoutFirstAutomaticSearch()
+        .init();
+    });
+
+    CommonFacetAssertions.assertDisplayFacet(SegmentedFacetSelectors, false);
+    CommonFacetAssertions.assertDisplayPlaceholder(
+      SegmentedFacetSelectors,
+      true
+    );
+  });
+
+  describe('with no label', () => {
+    function setupSegmentedFacetNoLabel() {
+      new TestFixture().with(addSegmentedFacet({field})).init();
+    }
+
+    describe('verify rendering', () => {
+      before(setupSegmentedFacetNoLabel);
+
+      CommonAssertions.assertAccessibility(segmentedFacetComponent);
+      CommonFacetAssertions.assertDisplayFacet(SegmentedFacetSelectors, true);
+      FacetAssertions.assertLabelExists(false);
+      FacetAssertions.assertNumberOfIdleBoxValues(defaultNumberOfValues);
+    });
+  });
 
-  describe.skip('with no label', () => {});
+  describe('when field returns no results', () => {
+    before(() => {
+      new TestFixture()
+        .with(addSegmentedFacet({field: 'notanactualfield', label}))
+        .init();
+    });
 
-  describe.skip('when field returns no results', () => {});
+    CommonFacetAssertions.assertDisplayFacet(SegmentedFacetSelectors, false);
+    CommonFacetAssertions.assertDisplayPlaceholder(
+      SegmentedFacetSelectors,
+      false
+    );
+    CommonAssertions.assertContainsComponentError(
+      SegmentedFacetSelectors,
+      false
+    );
+  });
 
-  describe.skip('with a selected path in the URL', () => {});
+  describe('with a selected path in the URL', () => {
+    before(() => {
+      new TestFixture()
+        .with(addSegmentedFacet({field, label}))
+        .withHash(`f[${field}]=Cervantes`)
+        .init();
+    });
 
-  describe.skip('with depends-on', () => {});
```

Comment:
Do we plan on supporting depends on?

Reply:
Not for now from my understanding. Will add them when we do.